website/docs/faq.md

```diff
@@ -353,3 +353,15 @@ a smooth transition to Ruby 3:
 For more information, watch [this section](https://youtu.be/2g9R7PUCEXo?t=2022)
 from Matz's RubyConf 2019 keynote, which talks about his plans for typing in
 Ruby 3.
+
+## Can I use Sorbet for duck typed code?
+
+No.
+
+Duck typed code relies on the partial interface of an object to determine if
```

Comment:
This would also be true about nominal types: you don't know exact value, but you know partial information about it - one of its superclasses.

WDYT about `Duck typing(or, more formally, Structural typing) specifies types by their structure: For example, ...`